python/paddle/jit/sot/opcode_translator/executor/variable_dispatch.py

```diff
@@ -645,16 +645,12 @@ def create_zip(*var: VariableBase):
 
 
 # map
-Dispatcher.register(
-    map,
-    (
-        "CallableVariable",
-        "VariableBase",
-    ),
-    lambda fn, var: MapVariable.from_iterator(
-        fn, var, graph=var.graph, tracker=DummyTracker([var])
-    ),
-)
+@Dispatcher.register_decorator(map)
+def create_map(func: CallableVariable, *var: VariableBase):
```

Comment:
```suggestion
def create_map(fn: CallableVariable, *vars: VariableBase):
```

后续我们统一使用 `fn` 而不是 `func`